tests/main/cgroup-tracking/task.yaml

```diff
@@ -0,0 +1,143 @@
+summary: Each snap app and hook is tracked via cgroups
+systems:
+    # Ubuntu 14.04's special version of systemd doesn't have StartTransientUnit API.
+    - -ubuntu-14.04-*
+    # TODO:UC20: figure out why dbus doesn't work for user sessions.
+    - -ubuntu-core-20-*
```

Comment:
Is this still a todo?

Reply:
Last time I looked yes. I will recheck today.

Reply:
This is now resolved.